building/build-debs/homeworld-admin-tools/src/seq.py

```diff
@@ -62,45 +64,84 @@ def ver():
 
 
 def sequence_core(ops: setup.Operations) -> None:
-    # ** Install and upgrade packages on all systems **
-    # spire infra install-packages
+    print("commands being executed to set up the core kubernetes cluster:")
+    print("  to install packages:")
+    print("    $ spire infra install-packages")
     infra.infra_install_packages(ops)
-
-    # ** Launch services **
-    # spire setup services
+    print("  to launch services:")
+    print("    $ spire setup services")
     setup.setup_services(ops)
-
-    # spire verify etcd
+    print("    $ spire verify etcd")
     ops.add_operation("verify that etcd has launched successfully",
                       iterative_verifier(verify.check_etcd_health, 20.0))
-    # spire verify kubernetes
+    print("    $ spire verify kubernetes")
     ops.add_operation("verify that kubernetes has launched successfully",
                       iterative_verifier(verify.check_kube_health, 10.0))
 
 
 def sequence_registry(ops: setup.Operations) -> None:
+    print("commands being executed to set up the bootstrap container registry:")
+    print("  to set up dns bootstrapping:")
+    print("    $ spire setup dns-bootstrap")
     setup.setup_dns_bootstrap(ops)
+    print("  to launch the registry:")
+    print("    $ spire setup bootstrap-registry")
     setup.setup_bootstrap_registry(ops)
+    print("  to confirm that the registry works:")
+    print("    $ spire verify aci-pull")
     ops.add_operation("verify that acis can be pulled from the registry", verify.check_aci_pull)
 
 
 def sequence_flannel(ops: setup.Operations) -> None:
+    print("commands being executed to set up flannel:")
+    print("  to deploy flannel:")
+    print("    $ spire deploy flannel")
     ops.add_operation("deploy or update flannel", lambda: deploy.launch_spec("flannel.yaml"))
+    print("  to verify flannel:")
+    print("    $ spire verify flannel-run")
     ops.add_operation("verify that flannel is online", iterative_verifier(verify.check_flannel_kubeinfo, 60.0))
+    print("    $ spire verify flannel-ping")
     ops.add_operation("verify that flannel is functioning", verify.check_flannel_function)
 
 
 def sequence_dns_addon(ops: setup.Operations) -> None:
+    print("commands being executed to set up dns-addon:")
+    print("  to deploy dns-addon:")
+    print("    $ spire deploy dns-addon")
     ops.add_operation("deploy or update dns-addon", lambda: deploy.launch_spec("dns-addon.yaml"))
+    print("  to verify dns-addon:")
+    print("    $ spire verify dns-addon-run")
     ops.add_operation("verify that dns-addon is online", iterative_verifier(verify.check_dns_kubeinfo, 60.0))
+    print("    $ spire verify dns-addon-query")
+    ops.add_operation("verify that dns-addon is functioning", verify.check_dns_function)
+
+
+def sequence_addons(ops: setup.Operations) -> None:
```

Comment:
Should reuse sequence_flannel and sequence_dns_addon as in sequence_supervisor.

Reply:
No -- that's purposeful. They should be interleaved for the sake of spending less time waiting for deployment.

Reply:
Alright, if that's intentional. It's minor anyway if the verification steps will ultimately be isolated as per #172.